package.json

```diff
@@ -11,6 +11,7 @@
     "cors": "^2.8.5",
     "crypto": "^1.0.1",
     "debug": "~2.6.9",
+    "dotenv": "^8.2.0",
```

Comment:
remove from package.json. Use [this](https://stackoverflow.com/a/13066677) as reference